app/Services/Model/AttendeeService.php

```diff
@@ -768,4 +770,60 @@ public function resynchAttendeesStatusBySummit(int $summit_id): void
 
         Log::debug(sprintf("AttendeeService::resynchAttendeesStatusBySummit summit id %s processed %s attendees", $summit_id, $count));
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function upsertAttendeeNote(Summit $summit, int $attendee_id, ?int $note_id, array $payload): SummitAttendeeNote {
```

Comment:
u need to pass the current member bc its the author
the author should be get from controller and pass it ot the service

Reply:
done